SPECS/azurelinux-repos/azurelinux-amd-preview.repo

```diff
@@ -0,0 +1,19 @@
+[azurelinux-official-amd-preview]
```

Comment:
I think we should also update the spec file to make this repo installable.

Reply:
I updated the signatures file...does that make it installable?

Reply:
No, the `azurelinux-repos.spec` file. Right now you've only added the files to the git repository but the spec doesn't install them or even know about them.

Reply:
thanks, now I got it building!